sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/PartitionProcessorFactoryImpl.java

```diff
@@ -60,20 +93,24 @@ public PartitionProcessor create(Lease lease, ChangeFeedObserver observer) {
             throw new IllegalArgumentException("lease");
         }
 
-        String startContinuation = lease.getContinuationToken();
-
-        if (startContinuation == null || startContinuation.isEmpty()) {
-            startContinuation = this.changeFeedProcessorOptions.getStartContinuation();
+        FeedRangeInternal feedRange = new FeedRangePartitionKeyRangeImpl(lease.getLeaseToken());
+        ChangeFeedState state;
+        if (Strings.isNullOrWhiteSpace(lease.getContinuationToken())) {
+            state = new ChangeFeedStateV1(
+                BridgeInternal.extractContainerSelfLink(this.collectionSelfLink),
+                new FeedRangePartitionKeyRangeImpl(lease.getLeaseToken()),
+                ChangeFeedMode.INCREMENTAL,
```

Comment:
We need to add a public facing configuration setting to toggle between full fidelity and incremental.

Reply:
Like discussed in the design meeting - not once you start processing for an existing continuation. You can always start a new Incremental change feed.
I simply don't see a scenario for this - either your app needs full fidelity information - in that case casting down to incremental would still break the app - or you don't need previous images in that case you can start-off with incremental (and it allows you to pick a starting point anyway).

I think we can reopen this discussion when looking at implementing the "IncrementalFullFidleity" mode Michael and company form the backend have in mind and the Compute team is pushing for. Matias and I have pushed back at adding this to public surface area because it is too confusing and we didn't have good enough understanding of possible use cases yet. But when we add that I can see how you would at least want to  fall back FullFidelity continuation to IncrementalFullFidelity in cases where your processor is out of retention window. 
Does that make sense?